dai/dsr-integration-guide/dsr-integration-guide-01.md

```diff
@@ -48,6 +52,42 @@ This guide will explain the Dai Savings Rate and how to integrate DSR into your
 
 - [Working seth](https://docs.makerdao.com/clis/seth)
 
+## Integration Stories
+
+As described in the rest of this guide, there are many approaches to integrate DSR support into your system. Each approach has pros and cons, varying levels of complexity, and different technical requirements. This section is purposed to help streamline the integration experience for integration stories of common intention. Akin to directions on a map, integration stories help guide the reader through a suggested path, but they do not prohibit exploration of other paths, which may lead to improved or more thoughtful integrations. Each story begins with some assumptions of the developer’s system, an introduction to the story, and the contents thereof.
+
+### Centralized Exchange
+
+The following assumptions about your system’s design and operation:
+Single cold/hot wallet(s) holding assets for multiple users
+Scalable off-chain accounting system holding a record of users’ balances
+Asynchronous contract calls following User action
+Ability to make calls to contracts that don’t inherit the ERC20 spec
+Will not need to interact with adjacent systems that require a [proxy identity](https://github.com/makerdao/developerguides/blob/master/devtools/working-with-dsproxy/working-with-dsproxy.md).
+
+With these assumptions, the following integration story is relieved of the proxy-identity requirement and prioritizes simplicity for the centralized exchange. In the following steps, we build up the prerequisite knowledge of the DSR and outline the steps to integrate the `DsrManager` before concluding with some recommendations. Considered as the simplest and most secure way of adding Dai to the DSR, the `DsrManager` is a smart contract that allows service providers to deposit/withdraw dai into the DSR contract ([Pot contract](https://docs.makerdao.com/smart-contract-modules/rates-module/pot-detailed-documentation)), and activate/deactivate the Dai Savings Rate to start earning savings on a pool of dai in a single function call. Let us begin:
```

Comment:
make sure to write "Dai" starting with capital D.

Reply:
Good catch! Thanks